src/client/debug/main/InteractiveFunction.svelte

```diff
@@ -69,10 +71,14 @@
 </style>
 
 <div class="move" class:active on:click={Activate}>
-  {name}(<span
+  <span>{name}</span>
+  <span>(</span>
+  <span
     class="arg-field"
     bind:this={span}
     on:blur={Deactivate}
+    on:keypress|stopPropagation={() => {}}
```

Comment:
First time reviewing Svelte code, but am I understanding correctly that the important change is this line preventing `on:keypress` listeners firing elsewhere while editing the contenteditable span and that that span is displayed conditionally somehow?